input_definition_test.go

```diff
@@ -226,6 +226,9 @@ func TestHandleAction(t *testing.T) {
 	value = true
 	action.ValueDestination = pilosa.InputSingleRowBool
 	b, err := pilosa.HandleAction(action, value, colID, timestamp)
+	if err != nil {
```

Comment:
@jaffee do we also need to handle the `err` on line 253?

Reply:
uh, yep, looks like it

Reply:
added it